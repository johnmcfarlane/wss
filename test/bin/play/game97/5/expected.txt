 63  1,9  | DAY
 49  1,8  | ADOR
 45  1,9  | DAO
 45  1,9  | DAR
 45  1,9  | DOA
 45  1,9  | DOR
 44  8,9  - ORYX
 42  3,9  | MAX
 40  1,8  | ADO
 36  1,9  | DA
 36  1,9  | DO
 34  1,8  | AD
 34  6,6  - COAXY
 34  6,6  - CORDAX
 32  5,3  - YOX
 31  5,3  - DOXY
 29  4,5  - AX
 29  5,3  - DOXA
 28  1,10 | AX
 28  1,10 | OX
 27  3,9  | MAY
 27  3,9  | MOY
 27  5,3  - ROXY
 25  2,10 - NAY
 25  2,10 - NOY
 24  3,1  | ARDU
 24  5,3  - ARX
 24  1,9  - DEMO
 24  3,9  | MAD
 24  3,9  | MOD
 24  3,1  | ORDU
 24  6,3  - ORYX
 24  5,3  - RAX
 24  5,3  - ROX
 24  3,1  | YARU
 23  8,9  - OARY
 22  6,6  - CORDY
 22  5,3  - ORYX
 22  6,3  - OXY
 22  8,7  - OXY
 22  8,9  - OXY
 21  5,3  - DORAY
 21  6,3  - DOXY
 21  3,1  | DROU
 21  3,9  | MAR
 21  3,9  | MOA
 21  3,9  | MOR
 21  5,1  | RAXED
 21  4,2  | XYZ
 20  6,2  | AXE
 20  5,2  | AXED
 20  4,3  | AZOX
 20  6,6  - CARRY
 20  8,9  - ORAD
 20  5,3  - YAD
 20  5,3  - YARD
 20  5,3  - YOD
 19  6,3  - AX
 19  8,7  - AX
 19  6,3  - OX
 19  8,7  - OX
 19  8,9  - OX
 19  5,2  | OXEA
 19  5,2  | OXER
 19  6,3  - ROXY
 19  5,3  | XED
 19  5,3  - YARR
 19  4,10 - YAS
 19  4,10 - YRS
 19  4,4  | ZAX
 18  5,2  | AXE
 18  6,6  - CARDO
 18  5,3  - DAY
 18  5,3  - DRY
 18  3,3  | XU
 18  5,3  - YAR
 18  5,3  - YOR
 17  4,5  - AD
 17  5,3  - DORY
 17  6,3  - DOXA
 17  5,3  - DRAY
 17  5,3  - ORARY
 17  8,9  - ORRA
 17  6,3  - YOX
 16  5,3  - ADY
 16  4,2  | DAZY
 16  4,2  | DOZY
 16  8,7  - ORYX
 16  4,1  | ORYZA
 16  5,3  - YA
 16  5,3  - YO
 15  5,3  - ADRY
 15  4,5  - AR
 15  5,3  - ARDOR
 15  5,3  - DARR
 15  5,3  - DORA
 15  5,3  - DORR
 15  3,2  | DOUAR
 15  5,3  | YEX
 14  6,3  - ADRY
 14  5,3  - ARY
 14  6,6  - COAX
 14  6,6  - COXA
 14  6,6  - CRAX
 14  5,3  - DAO
 14  5,3  - DAR
 14  5,3  - DOA
 14  4,6  | DOJO
 14  5,3  - DOR
 14  6,3  - DORY
 14  6,3  - DRAY
 14  4,8  | JODO
 14  4,8  | JOY
 14  5,3  - RAY
 14  4,2  | RAZOR
 14  5,3  - ROY
 14  5,3  - RYA
 14  6,10 - SOARY
 14  6,10 - SORRY
 14  6,1  | YADE
 14  6,3  - YARD
 14  8,4  | YARDS
 14  6,1  | YODE
 13  5,3  - ADOR
 13  4,2  | ADZ
 13  1,10 - ANY
 13  5,3  - ARRY
 13  6,3  - ARX
 13  8,7  - ARX
 13  1,10 | AY
 13  6,6  - COX
 13  5,3  | DEX
 13  6,3  - DORAY
 13  4,3  | DZO
 13  5,3  - OARY
 13  4,2  | ODZ
 13  1,10 - ONY
 13  1,10 | OY
 13  6,3  - RAX
 13  5,3  - RORY
 13  6,3  - ROX
 12  5,3  - ADO
 12  6,3  - ARRY
 12  5,3  - AY
 12  6,1  | AYRE
 12  4,3  | AZO
 12  5,3  - DA
 12  5,3  - DO
 12  8,4  | DRAYS
 12  8,4  | DRYAS
 12  3,3  | DUO
 12  6,2  | DYE
 12  5,3  - OAD
 12  6,3  - OARY
 12  5,3  - ODA
 12  5,3  - ORAD
 12  5,3  - ORD
 12  5,3  - OY
 12  5,3  - RAD
 12  5,3  - ROAD
 12  5,3  - ROD
 12  6,3  - RORY
 12  6,10 - SAROD
 12  6,10 - SORDA
 12  3,6  | YA
 12  6,1  | YARE
 12  6,3  - YARR
 12  3,6  | YO
 12  6,1  | YORE
 12  4,4  | ZOA
 11  8,7  - ADRY
 11  5,10 | AX
 11  1,5  | AY
 11  5,7  - DIA
 11  5,1  | DRYER
 11  6,3  - ORARY
 11  5,3  - ORRA
 11  5,10 | OX
 11  7,10 | OX
 11  5,1  | RAYED
 11  5,3  | REX
 11  5,3  - ROAR
 11  6,3  - YAD
 11  1,10 - YNO
 11  6,3  - YOD
 11  4,4  | ZA
 10  5,3  - AD
 10  1,10 | AD
 10  6,3  - ADOR
 10  6,3  - ADY
 10  8,7  - ADY
 10  5,3  - AOR
 10  6,3  - ARDOR
 10  5,3  - ARO
 10  5,3  - ARR
 10  2,6  - ARX
 10  6,2  | AYE
 10  6,6  - CADY
 10  3,6  | DA
 10  6,1  | DARE
 10  6,3  - DARR
 10  6,3  - DAY
 10  3,6  | DO
 10  6,3  - DORA
 10  6,3  - DORR
 10  8,4  | DORRS
 10  6,3  - DRY
 10  5,2  | DYER
 10  5,3  - OAR
 10  5,3  - OD
 10  1,10 | OD
 10  5,3  - ORA
 10  6,3  - ORAD
 10  1,6  - RAX
 10  6,3  - ROAD
 10  6,1  | RODE
 10  3,3  | RUA
 10  6,3  - RYA
 10  5,1  | RYDER
 10  6,2  | RYE
 10  8,8  | SAX
 10  6,10 - SAX
 10  6,10 - SORRA
 10  8,8  | SOX
 10  6,10 - SOX
 10  1,6  | YA
 10  6,3  - YAR
 10  6,3  - YOR
  9  1,5  | AD
  9  8,7  - ADOR
  9  8,3  | ARDORS
  9  8,7  - ARRY
  9  2,6  - AX
  9  6,3  - AY
  9  8,7  - AY
  9  3,2  | AYU
  9  6,6  - CARY
  9  6,6  - CORY
  9  6,6  - CRAY
  9  6,6  - CROY
  9  5,1  | DOREY
  9  5,2  | DYE
  9  5,4  | EX
  9  8,7  - OARY
  9  1,10 - ONA
  9  6,3  - OY
  9  8,7  - OY
  9  8,9  - OY
  9  5,2  | OYER
  9  4,6  - RACY
  9  6,3  - YA
  9  5,1  | YADE
  9  6,3  - YO
  9  5,1  | YODE
  8  5,6  - ACY
  8  6,3  - ADO
  8  8,7  - ADO
  8  5,3  - AR
  8  5,1  | ARDER
  8  6,3  - ARY
  8  8,7  - ARY
  8  5,2  | AYE
  8  6,6  - CARD
  8  6,6  - CAY
  8  6,6  - CODA
  8  6,6  - CORD
  8  6,6  - COY
  8  6,6  - CRY
  8  1,6  | DA
  8  6,3  - DAO
  8  6,3  - DAR
  8  5,6  - DCOR
  8  5,3  | DEY
  8  6,3  - DOA
  8  6,2  | DOE
  8  6,3  - DOR
  8  5,2  | DREY
  8  3,9  | MA
  8  3,9  | MO
  8  2,10 - NA
  8  2,10 - NO
  8  6,3  - ODA
  8  8,7  - ODA
  8  8,9  - ODA
  8  6,2  | ODE
  8  5,3  - OR
  8  8,7  - ORAD
  8  6,1  | ORAE
  8  5,1  | ORDER
  8  6,3  - ORRA
  8  6,1  | RARE
  8  6,3  - RAY
  8  8,4  | ROADS
  8  6,3  - ROAR
  8  8,6  | ROSARY
  8  6,3  - ROY
  8  5,2  | RYE
  8  5,1  | YARER
  8  5,3  | YEA
  8  5,3  | YER
  8  5,7  - YI
  8  5,7  | YO
  7  5,6  - ACOR
  7  6,3  - AD
  7  8,7  - AD
  7  4,6  - ADC
  7  1,10 | AR
  7  4,6  - ARCO
  7  5,1  | AYRE
  7  6,6  - CAD
  7  6,6  - CARO
  7  6,6  - CARR
  7  6,6  - CDR
  7  6,6  - COD
  7  6,6  - CORA
  7  6,6  - CORR
  7  6,3  - DA
  7  5,1  | DARER
  7  8,5  | DAYS
  7  5,6  - DCA
  7  6,3  - DO
  7  4,6  - DOC
  7  8,5  | DRYS
  7  8,5  | DYAS
  7  6,3  - OAD
  7  8,7  - OAD
  7  8,9  - OAD
  7  5,1  | OARED
  7  6,3  - OD
  7  8,7  - OD
  7  8,9  - OD
  7  5,2  | ODEA
  7  1,10 | OR
  7  4,6  - ORCA
  7  6,3  - ORD
  7  8,7  - ORD
  7  8,9  - ORD
  7  8,7  - ORRA
  7  6,3  - RAD
  7  5,6  - RCD
  7  8,4  | ROARS
  7  6,3  - ROD
  7  5,1  | RODE
  7  5,2  | ROEY
  7  8,8  | SODY
  7  6,10 - SODY
  7  1,6  - YARD
  7  5,1  | YARE
  7  5,3  | YE
  7  8,5  | YODS
  7  5,1  | YORE
  7  3,2  | YOU
  6  2,6  - ADY
  6  6,3  - AOR
  6  8,7  - AOR
  6  4,6  - ARC
  6  6,2  | ARE
  6  5,2  | ARED
  6  6,3  - ARO
  6  8,7  - ARO
  6  6,3  - ARR
  6  8,7  - ARR
  6  5,10 | AY
  6  6,6  - CAR
  6  6,6  - COR
  6  6,6  - CRO
  6  5,2  | DAER
  6  5,1  | DARE
  6  3,2  | DAU
  6  1,6  - DAY
  6  5,3  | DEA
  6  5,3  | DER
  6  5,7  - DI
  6  5,7  | DO
  6  5,2  | DOER
  6  8,5  | DORSA
  6  8,6  | DYS
  6  6,3  - OAR
  6  8,7  - OAR
  6  8,9  - OAR
  6  1,6  - OARY
  6  5,6  - OCA
  6  5,2  | ODE
  6  6,3  - ORA
  6  8,7  - ORA
  6  8,9  - ORA
  6  4,6  - ORC
  6  6,2  | ORE
  6  5,2  | ORED
  6  5,10 | OY
  6  7,10 | OY
  6  1,6  - RAYA
  6  8,5  | RAYS
  6  4,6  - ROC
  6  6,2  | ROE
  6  5,2  | ROED
  6  8,6  | ROSY
  6  8,5  | RYAS
  6  8,8  | SORY
  6  6,10 - SORY
  6  8,8  | SOYA
  6  6,10 - SOYA
  6  8,8  | SYD
  6  6,10 - SYD
  6  1,6  - YAD
  6  1,6  - YARR
  5  5,10 | AD
  5  8,5  | ADOS
  5  6,3  - AR
  5  8,7  - AR
  5  3,2  | ARU
  5  2,6  - ARY
  5  8,6  | AYS
  5  1,6  - DARR
  5  5,3  | DE
  5  5,2  | DOE
  5  8,5  | DORS
  5  8,6  | DOSA
  5  5,10 | OD
  5  7,10 | OD
  5  6,3  - OR
  5  8,7  - OR
  5  8,9  - OR
  5  5,1  | ORAE
  5  1,6  - RAAD
  5  1,6  - RADA
  5  8,5  | RADS
  5  5,1  | RARE
  5  1,6  - RAY
  5  5,3  | RED
  5  8,5  | RODS
  5  5,2  | ROER
  5  8,8  | SADO
  5  6,10 - SADO
  5  8,8  | SADR
  5  6,10 - SADR
  5  8,8  | SARD
  5  6,10 - SARD
  5  8,8  | SAY
  5  6,10 - SAY
  5  8,8  | SODA
  5  6,10 - SODA
  5  8,8  | SORD
  5  6,10 - SORD
  5  8,8  | SOY
  5  6,10 - SOY
  5  8,8  | SYR
  5  6,10 - SYR
  5  1,6  - YAR
  5  8,6  | YAS
  5  8,6  | YRS
  4  2,6  - ADO
  4  8,6  | ADS
  4  5,3  | AER
  4  5,10 | AR
  4  5,2  | ARE
  4  8,7  | ASOR
  4  5,10 - ASOR
  4  2,6  - AY
  4  1,6  - DAO
  4  1,6  - DAR
  4  8,6  | DAS
  4  8,6  | DOS
  4  8,7  | DSR
  4  5,10 - DSR
  4  1,6  - OAD
  4  8,5  | OARS
  4  8,6  | ODS
  4  5,3  | OER
  4  5,7  - OI
  4  5,7  | OO
  4  5,10 | OR
  4  7,10 | OR
  4  8,5  | ORAS
  4  5,2  | ORE
  4  8,7  | OSAR
  4  5,10 - OSAR
  4  1,6  - RAD
  4  1,6  - RARA
  4  5,3  | REA
  4  5,2  | ROE
  4  8,6  | ROSA
  4  8,8  | SAD
  4  6,10 - SAD
  4  8,8  | SOAR
  4  6,10 - SOAR
  4  8,8  | SOD
  4  6,10 - SOD
  4  8,8  | SORA
  4  6,10 - SORA
  4  1,6  - YA
  4  6,3  | YE
  3  2,6  - AD
  3  5,3  | AE
  3  1,5  - AL
  3  1,10 - AN
  3  2,6  - AOR
  3  2,6  - ARO
  3  2,6  - ARR
  3  8,6  | ARS
  3  1,6  - DA
  3  6,3  | DE
  3  5,4  | ED
  3  1,6  - OAR
  3  5,3  | OE
  3  1,10 - ON
  3  8,6  | ORS
  3  8,6  | RAS
  3  5,3  | RE
  3  8,6  | ROS
  3  8,8  | SAO
  3  6,10 - SAO
  3  2,7  - TA
  3  2,7  - TO
  2  1,6  - AA
  2  2,6  - AA
  2  6,3  | AE
  2  2,6  - AR
  2  8,7  | AS
  2  5,10 - AS
  2  1,7  - AT
  2  5,4  | EO
  2  5,4  | ER
  2  6,3  | OE
  2  8,7  | OS
  2  5,10 - OS
  2  6,3  | RE
  2  8,8  | SO
  2  6,10 - SO
  1  1,8  - At
