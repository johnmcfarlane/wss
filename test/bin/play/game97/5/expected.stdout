 63  1,9  | DAY
 45  1,9  | DOR
 44  8,9  - ORYX
 42  3,9  | MAX
 40  1,8  | ADO
 36  1,9  | DA
 36  1,9  | DO
 34  1,8  | AD
 31  5,3  - DOXY
 29  4,5  - AX
 28  1,10 | AX
 28  1,10 | OX
 27  3,9  | MAY
 25  2,10 - NAY
 24  1,9  - DEMO
 24  3,9  | MAD
 24  3,9  | MOD
 24  6,3  - ORYX
 24  5,3  - RAX
 22  5,3  - ORYX
 22  6,3  - OXY
 22  8,7  - OXY
 22  8,9  - OXY
 21  6,3  - DOXY
 21  3,9  | MAR
 21  3,9  | MOA
 21  3,9  | MOR
 21  5,1  | RAXED
 20  6,2  | AXE
 20  5,2  | AXED
 20  6,6  - CARRY
 20  8,9  - ORAD
 20  5,3  - YARD
 20  5,3  - YOD
 19  6,3  - AX
 19  8,7  - AX
 19  8,9  - AX
 19  6,3  - OX
 19  8,7  - OX
 19  8,9  - OX
 19  4,4  | ZAX
 18  5,2  | AXE
 18  5,3  - DAY
 18  5,3  - DRY
 18  3,3  | XU
 18  5,3  - YAR
 17  4,5  - AD
 17  5,3  - DORY
 17  5,3  - DRAY
 17  8,9  - ORRA
 16  4,2  | DOZY
 16  8,7  - ORYX
 16  5,3  - YA
 16  5,3  - YO
 15  4,5  - AR
 15  5,3  - ARDOR
 15  5,3  - DORR
 14  6,6  - COAX
 14  6,6  - COXA
 14  4,6  | DOJO
 14  5,3  - DOR
 14  6,3  - DORY
 14  6,3  - DRAY
 14  4,8  | JOY
 14  5,3  - RAY
 14  4,2  | RAZOR
 14  5,3  - RYA
 14  6,10 - SORRY
 14  6,3  - YARD
 14  8,4  | YARDS
 13  4,2  | ADZ
 13  1,10 - ANY
 13  1,10 | AY
 13  6,6  - COX
 13  5,3  | DEX
 13  4,3  | DZO
 13  1,10 | OY
 13  6,3  - RAX
 12  5,3  - ADO
 12  5,3  - AY
 12  4,3  | AZO
 12  5,3  - DA
 12  5,3  - DO
 12  8,4  | DRAYS
 12  3,3  | DUO
 12  6,2  | DYE
 12  5,3  - OAD
 12  5,3  - ODA
 12  5,3  - ORAD
 12  5,3  - OY
 12  5,3  - RAD
 12  5,3  - ROAD
 12  5,3  - ROD
 12  6,10 - SAROD
 12  3,6  | YA
 12  6,1  | YARE
 12  3,6  | YO
 12  6,1  | YORE
 12  4,4  | ZOA
 11  5,10 | AX
 11  7,10 | AX
 11  1,5  | AY
 11  5,1  | DRYER
 11  5,3  - ORRA
 11  5,10 | OX
 11  7,10 | OX
 11  5,1  | RAYED
 11  5,3  | REX
 11  5,3  - ROAR
 11  1,10 - YNO
 11  6,3  - YOD
 11  4,4  | ZA
 10  5,3  - AD
 10  1,10 | AD
 10  6,3  - ARDOR
 10  6,2  | AYE
 10  3,6  | DA
 10  6,1  | DARE
 10  6,3  - DAY
 10  3,6  | DO
 10  6,1  | DORE
 10  6,3  - DORR
 10  8,4  | DORRS
 10  6,3  - DRY
 10  5,2  | DYER
 10  5,3  - OAR
 10  5,3  - OD
 10  1,10 | OD
 10  5,3  - ORA
 10  6,3  - ORAD
 10  1,6  - RAX
 10  6,3  - ROAD
 10  6,1  | RODE
 10  6,3  - RYA
 10  6,2  | RYE
 10  8,8  | SAX
 10  6,10 - SAX
 10  8,8  | SOX
 10  6,10 - SOX
 10  1,6  | YA
 10  6,3  - YAR
  9  1,5  | AD
  9  8,3  | ARDORS
  9  2,6  - AX
  9  6,3  - AY
  9  8,7  - AY
  9  8,9  - AY
  9  6,6  - CORY
  9  5,2  | DYE
  9  5,4  | EX
  9  6,3  - OY
  9  8,7  - OY
  9  8,9  - OY
  9  5,2  | OYER
  9  4,6  - RACY
  9  6,3  - YA
  9  6,3  - YO
  8  6,3  - ADO
  8  8,7  - ADO
  8  8,9  - ADO
  8  5,3  - AR
  8  5,2  | AYE
  8  6,6  - CARD
  8  6,6  - CAY
  8  6,6  - CODA
  8  6,6  - CORD
  8  6,6  - COY
  8  6,6  - CRY
  8  1,6  | DA
  8  5,3  | DEY
  8  6,2  | DOE
  8  6,3  - DOR
  8  3,9  | MA
  8  3,9  | MO
  8  2,10 - NA
  8  2,10 - NO
  8  6,3  - ODA
  8  8,7  - ODA
  8  8,9  - ODA
  8  6,2  | ODE
  8  5,3  - OR
  8  8,7  - ORAD
  8  5,1  | ORDER
  8  6,3  - ORRA
  8  6,1  | RARE
  8  6,3  - RAY
  8  8,4  | ROADS
  8  6,3  - ROAR
  8  8,6  | ROSARY
  8  5,2  | RYE
  8  5,1  | YARER
  8  5,3  | YEA
  8  5,7  - YI
  8  5,7  | YO
  7  6,3  - AD
  7  8,7  - AD
  7  8,9  - AD
  7  1,10 | AR
  7  4,6  - ARCO
  7  6,6  - CAD
  7  6,6  - CARR
  7  6,6  - COD
  7  6,3  - DA
  7  5,1  | DARER
  7  8,5  | DAYS
  7  6,3  - DO
  7  4,6  - DOC
  7  8,5  | DRYS
  7  6,3  - OAD
  7  8,7  - OAD
  7  8,9  - OAD
  7  5,1  | OARED
  7  6,3  - OD
  7  8,7  - OD
  7  8,9  - OD
  7  5,2  | ODEA
  7  1,10 | OR
  7  4,6  - ORCA
  7  8,7  - ORRA
  7  6,3  - RAD
  7  5,1  | RARED
  7  8,4  | ROARS
  7  6,3  - ROD
  7  5,1  | RODE
  7  1,6  - YARD
  7  5,1  | YARE
  7  5,3  | YE
  7  8,5  | YODS
  7  5,1  | YORE
  7  3,2  | YOU
  6  4,6  - ARC
  6  6,2  | ARE
  6  5,10 | AY
  6  7,10 | AY
  6  6,6  - CAR
  6  6,6  - COR
  6  5,1  | DARE
  6  1,6  - DAY
  6  5,7  - DI
  6  5,7  | DO
  6  5,2  | DOER
  6  5,1  | DORE
  6  8,5  | DORSA
  6  6,3  - OAR
  6  8,7  - OAR
  6  8,9  - OAR
  6  5,6  - OCA
  6  5,2  | ODE
  6  6,3  - ORA
  6  8,7  - ORA
  6  8,9  - ORA
  6  4,6  - ORC
  6  6,2  | ORE
  6  5,10 | OY
  6  7,10 | OY
  6  1,6  - RAYA
  6  8,5  | RAYS
  6  4,6  - ROC
  6  6,2  | ROE
  6  8,6  | ROSY
  6  8,5  | RYAS
  6  8,8  | SOYA
  6  6,10 - SOYA
  5  5,10 | AD
  5  7,10 | AD
  5  8,5  | ADOS
  5  6,3  - AR
  5  8,7  - AR
  5  8,9  - AR
  5  8,6  | AYS
  5  5,3  | DE
  5  5,2  | DOE
  5  8,5  | DORS
  5  5,10 | OD
  5  7,10 | OD
  5  8,5  | ODAS
  5  6,3  - OR
  5  8,7  - OR
  5  8,9  - OR
  5  8,5  | RADS
  5  5,1  | RARE
  5  1,6  - RAY
  5  5,3  | RED
  5  8,5  | RODS
  5  8,8  | SARD
  5  6,10 - SARD
  5  8,8  | SAY
  5  6,10 - SAY
  5  8,8  | SODA
  5  6,10 - SODA
  5  8,8  | SORD
  5  6,10 - SORD
  5  8,8  | SOY
  5  6,10 - SOY
  5  1,6  - YAR
  4  2,6  - ADO
  4  8,6  | ADS
  4  5,10 | AR
  4  7,10 | AR
  4  5,2  | ARE
  4  2,6  - AY
  4  8,6  | DOS
  4  1,6  - OAD
  4  8,5  | OARS
  4  8,6  | ODS
  4  5,7  - OI
  4  5,7  | OO
  4  5,10 | OR
  4  7,10 | OR
  4  5,2  | ORE
  4  8,7  | OSAR
  4  5,10 - OSAR
  4  1,6  - RAD
  4  5,2  | ROE
  4  8,8  | SAD
  4  6,10 - SAD
  4  8,8  | SOAR
  4  6,10 - SOAR
  4  8,8  | SOD
  4  6,10 - SOD
  4  8,8  | SORA
  4  6,10 - SORA
  4  1,6  - YA
  4  6,3  | YE
  3  2,6  - AD
  3  5,3  | AE
  3  1,5  - AL
  3  1,10 - AN
  3  8,6  | ARS
  3  1,6  - DA
  3  6,3  | DE
  3  5,4  | ED
  3  1,6  - OAR
  3  5,3  | OE
  3  1,10 - ON
  3  8,6  | ORS
  3  8,6  | RAS
  3  5,3  | RE
  3  2,7  - TA
  3  2,7  - TO
  2  1,6  - AA
  2  2,6  - AA
  2  6,3  | AE
  2  2,6  - AR
  2  8,7  | AS
  2  5,10 - AS
  2  1,7  - AT
  2  5,4  | EO
  2  5,4  | ER
  2  6,3  | OE
  2  8,7  | OS
  2  5,10 - OS
  2  6,3  | RE
  2  8,8  | SA
  2  6,10 - SA
  2  8,8  | SO
  2  6,10 - SO
  1  1,8  - At
